src/org/labkey/test/util/WorkbookHelper.java

```diff
@@ -75,6 +78,16 @@ public int getWorkbookIdFromUrl(URL url)
         }
     }
 
+    public void deleteWorkbooksFromDataRegion(DataRegionTable dr)
+    {
+        int checked = dr.getCheckedCount();
+        dr.clickHeaderButton("Delete");
+        assertTrue(_test.acceptAlert().contains("Are you sure you want to delete the selected row" + (checked == 1 ? "" : "s") + "?"));
```

Comment:
Use `assertAlert` so that the failure message will include the actual alert text.
```suggestion
        assertAlert("Are you sure you want to delete the selected row" + (checked == 1 ? "" : "s") + "?");
```

Reply:
ok, done